definitions/npm/@testing-library/user-event_v13.x.x/flow_v0.104.x-/user-event_v13.x.x.js

```diff
@@ -0,0 +1,170 @@
+declare type TypeOpts = {|
+  skipClick?: boolean,
+  skipAutoClose?: boolean,
+  initialSelectionStart?: number,
+  initialSelectionEnd?: number,
+|};
+declare type TypeOptsDelay = {|
+  skipClick?: boolean,
+  skipAutoClose?: boolean,
+  delay: number,
+  initialSelectionStart?: number,
+  initialSelectionEnd?: number,
+|};
+
+/* Source for keyboard: https://github.com/testing-library/user-event/blob/main/src/keyboard/types.ts */
+declare type KeyboardKey = {|
+  code?: string,
+  key?: string,
+  /** Location on the keyboard for keys with multiple representation
+   * STANDARD = 0,
+   * LEFT = 1,
+   * RIGHT = 2,
+   * NUMPAD = 3
+   */
+  location?: 0 | 1 | 2 | 3,
+  keyCode?: number,
+  altGr?: boolean,
+  shift?: boolean,
+|};
+declare type BaseKeyboardOpts = {|
+  document?: Document,
+  autoModify?: boolean,
+  keyboardMap?: Array<KeyboardKey>,
+|};
+declare type KeyboardState = {|
+  pressed: Array<{|
+    keyDef: KeyboardKey,
+    unpreventedDefault: boolean,
+  |}>,
+  modifiers: {|
+    alt: boolean,
+    caps: boolean,
+    ctrl: boolean,
+    meta: boolean,
+    shift: boolean,
+  |},
+  activeElement: TargetElement | null,
+  carryValue?: string,
+  carryChar: string,
+|};
+
+declare type KeyboardOpts = BaseKeyboardOpts;
+declare type KeyboardOptsDelay = {|
+  ...BaseKeyboardOpts,
+  /** Delay between keystrokes */
+  delay: number,
+|};
+
+declare type TabUserOptions = {|
+  shift?: boolean,
+  focusTrap?: Document | Element,
+|};
+
+// As of Flow 0.134.x WindowProxy is any, which would annihilate all typechecking.
+declare type TargetElement = Element; /* | WindowProxy */
+
+declare type FilesArgument = File | File[];
+
+declare type UploadInitArgument = {|
+  clickInit?: MouseEvent$MouseEventInit,
+  changeInit?: Event,
+|};
+
+declare type ClickOptions = {|
+  skipHover?: boolean,
+  clickCount?: number,
+|};
```

Comment:
All those types should be declared inside the definition module scope, otherwise they are global.

Reply:
Done, thanks.